pkg/ipcache/ipcache.go

```diff
@@ -436,10 +437,6 @@ func (ipc *IPCache) UpsertLabels(prefix netip.Prefix, lbls labels.Labels, src so
 	ipc.UpsertMetadata(prefix, src, resource, lbls)
 }
 
-func (ipc *IPCache) RemoveLabels(cidr netip.Prefix, lbls labels.Labels, resource ipcacheTypes.ResourceID) {
-	ipc.RemoveMetadata(cidr, resource, lbls)
```

Comment:
I don't think we should do this because it is a public API which may get used in future versions of v1.13.